output.go

```diff
@@ -159,14 +178,29 @@ func (l LogType) Configure(opts LogOptions) (send.Sender, error) {
 		if opts.InMemoryCap <= 0 {
 			return nil, errors.New("invalid inmemory capacity")
 		}
-		sender, err = send.NewInMemorySender("jasper", send.LevelInfo{Default: level.Trace, Threshold: level.Trace}, opts.InMemoryCap)
+		sender, err = send.NewInMemorySender(DefaultLogName, send.LevelInfo{Default: level.Trace, Threshold: level.Trace}, opts.InMemoryCap)
 		if err != nil {
 			return nil, err
 		}
 	default:
 		return nil, errors.New("unknown log type")
 	}
 
+	var formatter send.MessageFormatter
+	switch opts.Format {
+	case LogFormatDefault:
+		formatter = send.MakeDefaultFormatter()
+	case LogFormatPlain:
+		formatter = send.MakePlainFormatter()
+	case LogFormatJSON:
+		formatter = send.MakeJSONFormatter()
+	default:
+		return nil, errors.New("unknown log format")
+	}
```

Comment:
should we pull this out into a method on the formatter type?

Reply:
I moved this to its own LogFormat method.